Quadrangle.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FiguresLR1
+{
+    class Quadrangle : BaseFigure
+    {
+        private protected int x1, y1; 
```

Comment:
Your code doesn't have an implementation to change coordinates if you need it. Read about get; set; and why do people use it